relay-prosperoconv/build.rs

```diff
@@ -0,0 +1,78 @@
+use std::fs;
+use std::process::Command;
+
+use dircpy::CopyBuilder;
+use tempfile::tempdir;
+
+fn main() {
+    println!("cargo:rerun-if-changed=build.rs");
+    println!("cargo:rerun-if-env-changed=RUSTFLAGS");
+    println!("cargo:rerun-if-changed=prosperoconv.version");
+
+    // Need to explicitly check the env var here since if the `--target` flag is used the build
+    // script will not receive the cfgs:
+    // https://doc.rust-lang.org/nightly/cargo/reference/config.html#buildrustflags
+    if !cfg!(sentry)
+        && !std::env::var("CARGO_ENCODED_RUSTFLAGS").is_ok_and(|v| v.contains("--cfg\u{1f}sentry"))
+    {
+        println!("cargo:warning=Sentry flag not set");
```

Comment:
I assume you had those for testing, I think we shouldn't have warnings in the normal build output.